src/shapes/itext.class.js

```diff
@@ -1,14 +1,4 @@
 (function() {
-
-  function parseDecoration(object) {
-    if (object.textDecoration) {
-      object.textDecoration.indexOf('underline') > -1 && (object.underline = true);
-      object.textDecoration.indexOf('line-through') > -1 && (object.linethrough = true);
-      object.textDecoration.indexOf('overline') > -1 && (object.overline = true);
-      delete object.textDecoration;
-    }
-  }
```

Comment:
this was meant to aid for changes between fabric 1.x and fabric 2.0
I think it can go away now.